src/LegendrePolynomials.jl

```diff
@@ -303,6 +404,72 @@ julia> collectPl(0.5, lmax = 4)
 """
 collectPl(x; lmax::Integer) = collect(LegendrePolynomialIterator(x, lmax))
 
+"""
+	collectPlm(x; n::Integer, lmax::Integer)
```

Comment:
should this 'n' be 'm' instead?

Reply:
It should; thanks.

The order of arguments was also wrong.